pkg/operator/k8sutil/pod.go

```diff
@@ -21,21 +21,37 @@ package k8sutil
 import (
 	"encoding/json"
 	"fmt"
+	"os"
 
 	"k8s.io/client-go/1.5/pkg/api"
 	unversionedAPI "k8s.io/client-go/1.5/pkg/api/unversioned"
 	"k8s.io/client-go/1.5/pkg/api/v1"
 )
 
 const (
-	AppAttr     = "app"
-	ClusterAttr = "rook_cluster"
-	VersionAttr = "rook_version"
-	PodIPEnvVar = "ROOKD_PRIVATE_IPV4"
+	AppAttr           = "app"
+	ClusterAttr       = "rook_cluster"
+	VersionAttr       = "rook_version"
+	PodIPEnvVar       = "ROOKD_PRIVATE_IPV4"
+	DefaultRepoPrefix = "quay.io/rook"
+	RepoPrefixEnvVar  = "ROOK_OPERATOR_REPO_PREFIX"
```

Comment:
Nit: Instead of ROOK_OPERATOR_REPO_PREFIX, i would use a more exact var. How about REGISTRY_LOCATION? This can be used not just for rook operator but also for the other images like mon, osd etc